runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPlugin.java

```diff
@@ -143,6 +143,11 @@
 	// Chest loot handling
 	private static final String CHEST_LOOTED_MESSAGE = "You find some treasure in the chest!";
 	private static final Pattern LARRAN_LOOTED_PATTERN = Pattern.compile("You have opened Larran's (big|small) chest .*");
+
+	private static final String COFFIN_LOOTED_MESSAGE = "You push the coffin lid aside.";
+	private static final int HALLOWED_SEPULCHRE_COFFIN_REG = 39535;
```

Comment:
This shouldn't be hardcoded, thought I'm also not sure if it should be referencing the NullObjectID list either. Not sure why the coffin is a null object anyway.

Reply:
Interesting, Im not aware the difference between Object/NullObject. I was wondering why I couldn't find the coffin's # in the ObjectID class though. So that explains it

I tried to use RegionIds like the plugin is currently using for Chest events like Larrans, Crystal, Brimstone, etc. The Sepulchre must be shared instanced or something? though because they would change every run. Not sure how to go about handling this, this is first time I've ever done anything with RL code. 

I will properly at least reference it from NullObjectID in the meanwhile?

Reply:
So I looked into how Kraken was operating and using DevOptions grabbed a Map region # from each floor. Using the Location Plugin in dev plugins. Did 3 runs and took a new screenshot eachtime per floor to make sure the Map region number was reliable. 

I can only reach the 3rd floor though, someone will have to find numbers for the 4th and fifth